Rakefile

```diff
@@ -0,0 +1,3 @@
+require 'sinatra/activerecord/rake'
+
+require_relative 'environment'
```

Comment:
Rakefile not currently doing anything. Is this where we should run seeds.rb or similar?